docker/mining.sh

```diff
@@ -18,8 +18,10 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 source /src/docker/utils.sh
-ssh_check
-dvc_pull_models
+# Commented to allow the use of a DVC remote from a different type than SSH.
+# ssh_check
+# Not usable in README as it works only when inside the `bbs_` containers.
+# dvc_pull_models
```

Comment:
Could you please remind me how the server is going to find the NER models now?

Reply:
The decision taken by the team yesterday to comment these lines puts the mining server in the same situation of the search server: `dvc pull` needs to have been run before. In the README, there is already a `dvc pull` happening. Commenting the to lines doesn't therefore break something. For further discussions on the subject, there is the ticket [BBS-306](https://bbpteam.epfl.ch/project/issues/browse/BBS-306).

Reply:
Yes, I was trying to make sure I understand the setting of the path to the models correctly. The mining server finds the models by reading the `BBS_DATA_AND_MODELS_DIR` environment variable. Here's my current understanding of setting this variable, could you maybe please check if it's correct?

With `dvc_pull_models` the models would be pulled somewhere locally on the container, so that the value of this variable needs to be configured to point to a local path, e.g.:
```
BBS_DATA_AND_MODELS_DIR=/src/data_and_models
```

After commenting out `dvc_pull_models` the models will be provided externally. So we need to `docker run` with a mount of the volume where these models are, e.g. with the `-v /raid:/raid` flag, and then configure the environment variable to point to a location on that volume:
```
BBS_DATA_AND_MODELS_DIR=/raid/<...>/projects/search/data_and_models
```
Is this understanding correct?

Reply:
Yes.